go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20200413115906-b5235f65be36 // indirect
-	google.golang.org/grpc v1.26.0
+	google.golang.org/grpc v1.27.0
```

Comment:
this grpc update seems unrelated?

Reply:
Oh, It seems that I skipped `go mod verify` in the go-ceph version update. Thanks!

Reply:
I changed this to the go-ceph's version change.

Reply:
I am still confused by this... If there is a grpc update, those files should get included in `vendor/` too, but I do not see them. go-ceph does not depend on grpc, so I do not know where it comes from.

Ideally you would not include a grpc update in here at all, or maybe as a 3rd commit (1st in the series)?